src/inflators/text.ts

```diff
@@ -0,0 +1,120 @@
+import { addComponent } from "bitecs";
+import { BackSide, DoubleSide, FrontSide } from "three";
+import { Text as TroikaText } from "troika-three-text";
+import { HubsWorld } from "../app";
+import { TextTag } from "../bit-components";
+import { addObject3DComponent } from "../utils/jsx-entity";
+
+export type TextParams = {
+  value: string;
+  anchorX?: "left" | "center" | "right";
+  anchorY?: "top" | "top-baseline" | "top-cap" | "top-ex" | "middle" | "bottom-baseline" | "bottom";
+  clipRect?: [number, number, number, number] | null;
+  color?: string;
+  curveRadius?: number;
+  depthOffset?: number;
+  direction?: "auto" | "ltr" | "trl";
+  fillOpacity?: number;
+  fontUrl?: string | null;
+  fontSize?: number;
+  letterSpacing?: number;
+  lineHeight?: number | "normal";
+  maxWidth?: number;
+  opacity?: number;
+  outlineBlur?: number | `${number}%`;
```

Comment:
I've never seen this type definition before. What does `${number}%` mean?

Reply:
It might be similar to regex. (If I understand correctly) `${number}%` means number string that ends with "%", like "25%" or "0.5%".

https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html#handbook-content
https://stackoverflow.com/questions/51445767/how-to-define-a-regex-matched-string-type-in-typescript

Reply:
TIL. Thanks!

Reply:
Didn't know that was supported either. Neat!